alicja/config/proto/interface.proto

```diff
@@ -9,13 +9,13 @@ option go_package = "github.com/googleinterns/step224-2020/config/proto"
 import "github.com/googleinterns/step224-2020/config/proto/targets.proto";
 
 service Hermes_Ceph {
```

Comment:
This config won't be unique to Ceph, please rename it to something else, maybe HermesProberService or HermesProber?

Reply:
Sorry I thought I did rename it! Changed the name now to HermesProber